static/app/views/replays/deadRageClick/deadRageSelectorCards.tsx

```diff
@@ -1,97 +1,264 @@
+                  />
+                ),
+              };
+            })}
+          />
+        </StyledContentContainer>
+      )}
+      <SearchButton
+        label={t('See all selectors')}
+        path="selectors"
+        sort={`-${clickType}`}
+      />
+    </WidgetContainer>
   );
 }
 
-function RageClickTable({location}: {location: Location<any>}) {
-  const {isLoading, isError, data} = useDeadRageSelectors({
-    per_page: 4,
-    sort: '-count_rage_clicks',
-    cursor: undefined,
-    prefix: 'selector_',
-    isWidgetData: true,
-  });
+function AccordionItemHeader({
+  count,
+  deadOrRage,
+  selector,
+}: {
+  count: number;
+  deadOrRage: 'dead' | 'rage';
+  selector: string;
+}) {
+  const clickCount =
+    deadOrRage === 'dead' ? (
+      <DeadClickColor>
+        <IconCursorArrow size="xs" />
+        {count}
+      </DeadClickColor>
+    ) : (
+      <RageClickColor>
+        <IconCursorArrow size="xs" />
+        {count}
+      </RageClickColor>
+    );
 
   return (
-    <SelectorTable
-      data={data.filter(d => (d.count_rage_clicks ?? 0) > 0)}
-      isError={isError}
-      isLoading={isLoading}
-      location={location}
-      clickCountColumns={[{key: 'count_rage_clicks', name: 'rage clicks'}]}
-      title={
-        <Fragment>
-          <IconContainer>
-            <IconCursorArrow size="xs" color="red300" />
-          </IconContainer>
-          {t('Most Rage Clicks')}
-        </Fragment>
-      }
-      customHandleResize={() => {}}
-      clickCountSortable={false}
-    />
+    <StyledAccordionHeader>
+      <TextOverflow>
+        <code>{selector}</code>
+      </TextOverflow>
+      <RightAlignedCell>{clickCount}</RightAlignedCell>
+    </StyledAccordionHeader>
+  );
+}
+
+function SearchButton({
+  label,
+  sort,
+  path,
+  ...props
+}: {
+  label: ReactNode;
+  path: string;
+  sort: string;
+} & Omit<ComponentProps<typeof LinkButton>, 'size' | 'to'>) {
+  const location = useLocation();
+  const organization = useOrganization();
+
+  return (
+    <StyledButton
+      {...props}
+      size="xs"
+      to={{
+        pathname: normalizeUrl(`/organizations/${organization.slug}/replays/${path}/`),
+        query: {
+          ...location.query,
+          sort,
+          query: undefined,
+          cursor: undefined,
+        },
+      }}
+    >
+      {label}
+    </StyledButton>
   );
 }
 
 const SplitCardContainer = styled('div')`
   display: grid;
   grid-template-columns: 1fr 1fr;
-  grid-template-rows: max-content max-content;
+  grid-template-rows: max-content;
   grid-auto-flow: column;
   gap: 0 ${space(2)};
   align-items: stretch;
   padding-top: ${space(1)};
 `;
 
-const IconContainer = styled('span')`
-  margin-right: ${space(1)};
+const DeadClickColor = styled(TextOverflow)`
+  color: ${p => p.theme.yellow300};
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  gap: ${space(0.5)};
+  align-items: center;
+`;
+
+const RageClickColor = styled(TextOverflow)`
```

Comment:
We could combine `<DeadClickColor>` and `<RageClickColor>` by allowing color as a prop. Then instead of passing `deadOrRage` as a prop higher up, we just pass the color name itself, using ColorOrAlias again.


```diff
<AccordionWidget
+   color="red300"
-   deadOrRage="dead"
```

```
import {ColorOrAlias} from 'sentry/utils/theme';

const ClickColor = styled(TextOverflow)<{color: ColorOrAlias}>`
  color: ${p => p.color};
  ...more css...
`;
```